_posts/jasonwang/2016-02-03_jasonwang- Chapter_5_Exercises.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: wagerpascal
+title: "The Welcome to List City Sign"
+---
+
+Exercise 1:
+<iframe src="https://trinket.io/embed/python/04b6c55ee4" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I initially was attempting to solve the problem without using lists, but broke from the temptation and decided to use them. I knew that Matlab had a method in which you could concatenate additional objects to a matrix, so I investigated the python version, which is the MatrixName.extend() function. 
+Mildly surprised that I can simply state what things I should concatenate onto my matrix through a function, rather than the Matlab way.
+I also discovered the "sum" function for lists, which greatly simplified how I would go about calculating the totals.
+For the "count", a simple counter was established. Not really intended as most counters, but still good.
+
+Exercise 2:
+<iframe src="https://trinket.io/embed/python/9ee76e9925" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I realized I could use most of the same code from the first exercise. I actually accidentally discovered the max and min functions - I was first attempting to name the variable "max" and "min", but I noticed the different font colors.
+The max() and min() functions also work the same method as the sum() function, while the calculation of the average remained the same.
+
+At least I have a newfound appreciation of lists now.
```

Comment:
max and min loop through a list and remember the max and min- how would you remember the max and min as you're loping through the user's input?  The instinct to use built-in tools is good but again make sure you understand the fundamentals here.